onnx_tf/handlers/backend/sequence_erase.py

```diff
@@ -0,0 +1,43 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("SequenceErase")
+class SequenceErase(BackendHandler):
+
+  @classmethod
+  def chk_pos_in_bounds(cls, input_seq, pos):
+    """
+    Check the position is in-bounds with respect to the sequence.
+    Accepted range for 'position' is in [-n, n - 1], where n is the
+    number of tensors in 'input_sequence'.
+
+    :param input_seq: input sequence
+    :param pos: position of the output tensor
+
+    :return: True if position is in-bounds 
+    """
+    seq_length = tf.shape(input_seq.to_sparse())[0]
+    pos = tf.cast(pos, tf.int32)
+    cond1 = tf.greater_equal(pos, tf.negative(seq_length))
+    cond2 = tf.less_equal(pos, seq_length - 1)
+
+    # pos >= -n and pos < n
+    return tf.reduce_all(tf.logical_and(cond1, cond2))
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    input_sequence = tensor_dict[node.inputs[0]]
+    position = tensor_dict[node.inputs[1]]
```

Comment:
position is an optional input, need to set a default value if not set by user

Reply:
Updated with same code in PR https://github.com/onnx/onnx-tensorflow/pull/658